CI/scripts/algorithm_names_map.dat

```diff
@@ -0,0 +1,52 @@
+# #############################################################################
+# Certifier Framework Encryption algorigthm names mapping
+# Specify the way hard-coded strings will be remapped based on some
+# organization.
+# #############################################################################
+#
+# enc_method:Encryption algorithms supported.
```

Comment:
This phrase: `Encryption algorithms supported.` will come out in comments.

Would like to get clearer descriptions / explanations, so lay-people can on-board easily with some help from comments.